source/features/sticky-sidebar.tsx

```diff
@@ -35,9 +36,7 @@ void features.add(__filebasename, {
 	additionalListeners: [
 		() => {
 			window.addEventListener('resize', onResize);
```

Comment:
This should also be dropped

Reply:
I think this is needed, because the window can get resized and become too small to fit the whole sidebar without the dimensions of the sidebar element actually changing.

Reply:
Indeed. You might want to move this to init though. I’m not sure of why I put it in additionalListeners like this

Reply:
I am not at a computer, but if I remember correctly it was put by additional listeners since it needs to run on new comments. (The side bar gets refreshed or something like that)
I can be totally wrong.
Feel free to mark this as resolved.

Reply:
> I am not at a computer

Yes but [are you a yellow smiley](https://www.youtube.com/watch?v=lGOofzZOyl8)?

Reply:
> it was put by additional listeners since it needs to run on new comments

I didn't check but I think I put it there to avoid creating an init function, which was kinda silly. `additionalListener` is useful because get passed the `init` and then the listener calls it, but in this case neither one does that 🤷‍♂️

Either way the new comments are now handled by the observer